### DISCUSSION TITLE
How do I get the tabs to scroll directly to the active tab?

### DISCUSSION THREAD

**Discussion by mochekaramadan** [quasarframework/quasar#10238] [CLI - PWA mode] [answered]
Initial display when the page is first opened
![image](https://user-images.githubusercontent.com/30065254/127502715-ef2cf4c4-5a85-4509-a370-73f0fde2ea99.png)

I want the tabs to scroll automatically the first time the page is opened
![image](https://user-images.githubusercontent.com/30065254/127502768-a2cccd46-bad4-4bcb-89f6-893d75cbba56.png)

Note: I used Qtab

**Comment by pdanpdan** [comment, marked as answer]
duplicate of https://github.com/quasarframework/quasar/discussions/10034

**Reply by mochekaramadan** [reply]
I have tried the solution from the post link. the scroll works but the scroll still doesn't lead to the active tab 

I have 31 tabs when the page is first run, the scroll only leads to tabs between numbers 7 to 22 not showing the active tab, which is tab 29

**Reply by pdanpdan** [reply]
As mentioned în that thread, it's not in the official build yet - you need to use my builds - https://github.com/quasarframework/quasar/discussions/10034#discussioncomment-1014454

Just check on the releases page and use latest release version.
